ubuntu_bug_triage/triage.py

```diff
@@ -98,12 +99,21 @@ def updated_bugs(self):
         for bug_id in sorted(self._tasks_to_bug_ids(updated_tasks)):
             bug = Bug(self.launchpad.bugs[bug_id])
 
+            lpbug = self.launchpad.bugs[bug_id]
+            commenter = None
+            for comment in lpbug.messages:
+                commenter = comment.owner_link.split('~')[1]
+            person = None
+            for active in lpbug.activity:
+                person = active.person_link.split('~')[1]
+
             if self.team.name in BLACKLIST:
                 if self._all_src_on_blacklist(bug.tasks, self.team.name):
                     self._log.debug("skipping bug: %s", bug_id)
                     continue
 
-            bugs.append(bug)
+            if commenter not in self.ignore and person not in self.ignore:
+                bugs.append(bug)
```

Comment:
I know the script is slow as it is, but we should see what the impact is to check this, even when someone does not set the ignore flag. If it makes a difference we may want to check the ignore flag first.

Reply:
Right, I've tested on the desktop-packages set over a day, which had 73 entries. The runtime went from 15 seconds to 30 seconds, which is non trivial. I've updated the code now to only do the work if the ignore-user list is not empty